### ISSUE TITLE
Implement some way to handle multiple types of 'contents'

### ISSUE THREAD

**Issue by LogicalError** [LogicalError/realtime-CSG-for-unity#206]
Right now realtime-CSG assumes all brushes are solid. But it would make sense to create shapes out of, for instance, water or glass using brushes. But these brushes shouldn't remove the polygons inside them

**Comment by Petethegoat** [comment]
This is also useful for creating colliders/invisible walls etc. 👍

**Comment by LogicalError** [comment]
True! good point, I hadn't considered that situation .. that makes this even more useful
